charts/stable/kitchenowl/values.yaml

```diff
@@ -82,13 +82,13 @@ workload:
           imageSelector: backendImage
           env:
             FRONT_URL: "{{ .Values.workload.main.podSpec.containers.main.env.FRONT_URL }}"
-            OIDC_ISSUER: ""
-            OIDC_CLIENT_ID: ""
-            OIDC_CLIENT_SECRET: ""
-            APPLE_CLIENT_ID: ""
-            APPLE_CLIENT_SECRET: ""
-            GOOGLE_CLIENT_ID: ""
-            GOOGLE_CLIENT_SECRET: ""
+            OIDC_ISSUER: "{{ .Values.workload.main.podSpec.containers.main.env.OIDC_ISSUER }}"
+            OIDC_CLIENT_ID: "{{ .Values.workload.main.podSpec.containers.main.env.OIDC_CLIENT_ID }}"
+            OIDC_CLIENT_SECRET: "{{ .Values.workload.main.podSpec.containers.main.env.OIDC_CLIENT_SECRET }}"
+            APPLE_CLIENT_ID: "{{ .Values.workload.main.podSpec.containers.main.env.APPLE_CLIENT_ID }}"
+            APPLE_CLIENT_SECRET: "{{ .Values.workload.main.podSpec.containers.main.env.APPLE_CLIENT_SECRET }}"
+            GOOGLE_CLIENT_ID: "{{ .Values.workload.main.podSpec.containers.main.env.GOOGLE_CLIENT_ID }}"
+            GOOGLE_CLIENT_SECRET: "{{ .Values.workload.main.podSpec.containers.main.env.GOOGLE_CLIENT_SECRET }}"
```

Comment:
No please. I'd prefer making the backend the main workload and adjusting the services to target the correct workload/container.